### ISSUE TITLE
[BUG]can't get the model via the URL

### ISSUE THREAD

**Issue by leonodelee** [TharinduDR/TransQuest#19]
**Describe the bug**
I can't get your model via your url in https://tharindudr.github.io/TransQuest/models/sentence_level_pretrained/ ; It's said "The connection has been reset"

**Expected behaviour**
I want to get your model

**Comment by TharinduDR** [comment]
What is the model you are trying to get?

**Comment by leonodelee** [comment]
> What is the model you are trying to get?

I want to apply your en2zh model to my own data.

**Comment by leonodelee** [comment]
I wish with this model, I can  do Translation Quality Estimation on my dataset or input

**Comment by TharinduDR** [comment]
There are two models for en-zh, direct assessment and HTER prediction, What is the model you want to use?

**Comment by leonodelee** [comment]
> There are two models for en-zh, direct assessment and HTER prediction, What is the model you want to use?

Can I get both of them， QWQ; since I am not familiar with them yet.

**Comment by TharinduDR** [comment]
If you are not able to access the drive you should be able to find them in HuggingFace model hum - https://huggingface.co/TransQuest

**Comment by leonodelee** [comment]
Could you please give me the model of MTransQuest⊗- Aug on enzh language pair, there only MTransQuest-base in the HuggingFace model.

**Comment by leonodelee** [comment]
Could you please give me the model of MTransQuest⊗- Aug on enzh language pair, there only MTransQuest-base in the HuggingFace model.

**Comment by TharinduDR** [comment]
We are not releasing the MTransQuest⊗- Aug model because it has been largely finetuned to WMT 2020 dataset and it is not stable for other data.

**Comment by leonodelee** [comment]
Well, I just want to improve the accuracy of model scoring. I found that when apply the existing "monotransquest-hter-en_zh-wiki"  model, the test scores are concentrated in the range of 50-80. Some low score data in the training set are hit high in the test set, while the high score data in the training set are hit low. 
It is possible that the data precision is lost due to a series of operations such as normalization and de normalization in the process of training and testing; How can I deal with it and disperse it to make the score more reasonable?

**Comment by TharinduDR** [comment]
There are two reasons why I performed normalisation and denormalisation

1. It helped in the data augmentation process as mentioned in the paper. Also, it helped to interpret the output of the model. 
2. The other architecture - SiameseTransQuest needs to have labels in the 0-1 range as we consider cosine similarity as the loss function. Inorder to make the TransQuest framework unified, I took the decision to normalise the input to MonoTransQuest too. 

However, I don't think that if you don't normalise the scores, accuracy will be improved a lot. I have not checked it. But if you think like that, you can disable the normalisation and denormalisation in the training script and train the model again; compare the results

**Comment by leonodelee** [comment]
Mm-hmm. thank you for your reply. I just want to say that the  "monotransquest-hter-en_zh-wiki"  model predicting score seems to focus on the 0.5-0.8 range. For example, in the following example of training set, The prediction score is about 52, which is larger than their original average score of 38; There are also some examples where the average score in the training set is more than 90, but the score predicted by the model is only about 80. How can I improve this situation?

0	The last conquistador then rides on with his sword drawn.	最后的征服者骑着他的剑继续前进.	[34, 38, 42]	38.0	[-1.6082034156434533, -1.7847572356764654, -1.1493974315381452]	-1.514119360952688	-0.6990631818771362

**Comment by TharinduDR** [comment]
is this the dataset from Direct Assessment? Task 1: Sentence-Level Direct Assessment - http://www.statmt.org/wmt20/quality-estimation-task.html

**Comment by leonodelee** [comment]
Yes , it's from TransQuest-master/examples/sentence_level/wmt_2020/en_zh/data/en-zh/train.enzh.df.short.tsv

**Comment by TharinduDR** [comment]
Please have a look at the documentation - http://tharindu.co.uk/TransQuest/models/sentence_level_pretrained/. If you are looking at the DA task then you need the DA model - monotransquest-da-en_zh-wiki

**Comment by leonodelee** [comment]
I used to try it as its Pearson Correlation are similar with the un-DA but I havn't looked into the prediction, thank you for your guide, I will do experients to verify it.

**Comment by leonodelee** [comment]
Thank you for your kindness help.

**Comment by leonodelee** [comment]
Thank you for your kindness help.
